easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-2.10.9-goimpi-2020b.eb

```diff
@@ -0,0 +1,104 @@
+easyblock = 'Bundle'
```

Comment:
@fizwit please fix the type in the toolchain in the filename - should be `gompi`.

Reply:
typo fixed

Reply:
Nice that I included a typo in my request to fix a typo...